content/md/permissions-settings/manage-the-web-api-permissions_fr.md

```diff
@@ -0,0 +1,91 @@
+---
+id: manage-the-web-api-permissions
+themes: permissions-settings
+title: Gérer les permissions de l'**API Web**
+popular: false
+ee-only: false
+related: build-your-user-roles, what-is-a-role, what-is-a-user, manage-the-interface-and-actions-accesses
+---
+
+# Définir les autorisations sur l'API Web
```

Comment:
Finally, I think it's better if we harmonize by always using "permissions", instead of "autorisations".
```suggestion
# Définir les permissions sur l'API Web
```